### ISSUE TITLE
Trivial Pursuit Live! 2 [0100868013FFC000]

### ISSUE THREAD

**Issue by b12e** [Ryujinx/Ryujinx-Games-List#4297]
## Game Name
Trivial Pursuit Live! 2

#### Game Update Version: *(e.g. 1.0.0 or 1.1.3)*
00.01 and 01.02

#### Emulator Build Version: *(e.g. 1.1.373)*
1.1.478

#### Graphics Backend: *(e.g. Vulkan or OpenGL)*
Vulkan

#### Hardware Specs: 
Steam Deck (But also has the same issue described below on a 16" Intel Macbook Pro 2019 with i7 and AMD GPU)

*Describe the current game state here. 
List the issues you've noticed if there are any. 
Add links to related issues from the main repository if they exist.
Add links to pull requests that fix issues described if/when applicable. 
Add a zipped save file for easy access to any issues if needed, namely for situations that can only be accessed after several hours of gameplay.*
The game used to render blurry text. This is now fixed, but it doesn't get past the initial input screen (press A / B depending on input mapping to start) - it looks like it's not receiving any input.

#### Screenshots
<img width="1288" alt="image" src="https://user-images.githubusercontent.com/1340823/208323950-5bcfabd7-af12-459e-83cb-ace9f19c0840.png">

**Comment by MetrosexualGarbodor** [comment]
Missing a log file. Also are you completely sure your controller is set up correctly?

**Comment by b12e** [comment]
Yeah don't have the log for Steam Deck. Was trying to get it and update it for Mac OS.

The first version of the game works perfectly (Trivial Pursuit Live!) with the controller as set up right now.

**Comment by EmulationFanatic** [comment]
These games, along with the Jackbox party pack, rely on guest Internet access to be working in order for controller input to be recognized. As far as I know, guest Internet access is broken in these games on the macOS build.

**Comment by b12e** [comment]
I enabled Guest Internet Access on Steam Deck and same issue. 

Also does not recognize touch input (clicking the Start Game button), it recognizes there is a mouse (and changes the interface to accommodate for a mouse) but doesn't click.

Let me try to get a log file on the steam deck.

**Comment by b12e** [comment]
Just wanted to update that the same game works on Yuzu without issues - Just updated Yuzu's firmware to 15.0.0 as well and I got past the initial "start game" screen.
